eng/pipelines/templates/stages/platform-matrix.json

```diff
@@ -1,6 +1,8 @@
 {
   "displayNames": {
-    "--disablecov": ""
+    "--disablecov": "",
+    "false": "",
```

Comment:
I'm not sure about this true/false display name mapping strategy. It feels a little odd but at lease in this repo they both map to empty.

Reply:
I think this whole area needs a revisit. The pattern I'm finding myself falling into to support a lot of these mode switches (generate coverage, test samples, etc.) is at odds with display name generation (unless we don't mind random `true/false` strings in the names), and also is starting to pull a little too much config out of the matrix. I don't have any great ideas yet.